### PR TITLE
Update if same

### PR DESCRIPTION
**Pull request by bserdar**

### CODE REVIEW

File: crud/src/main/java/com/redhat/lightblue/crud/CRUDOperationContext.java [hunk 1]
```diff
@@ -53,6 +53,8 @@
     private final CRUDOperation CRUDOperation;
     private final HookManager hookManager;
     private final ExecutionOptions executionOptions;
+    private final Set<String> documentVersions=new HashSet<>();
+    private boolean updateIfSame;
```
Comment (paterczm) [inline crud/src/main/java/com/redhat/lightblue/crud/CRUDOperationContext.java]: I would call it updateIfCurrent or updateIfNotStale. Same is not as clear to me.

File: crud/src/main/resources/json-schema/response.json [hunk 2]
```diff
@@ -69,7 +91,7 @@
                 }
             }
         },
-        "otherErrors": {
+        "errors": {
```
Comment (paterczm) [inline crud/src/main/resources/json-schema/response.json]: What is this change about?

File: crud/src/main/java/com/redhat/lightblue/crud/CRUDOperationContext.java (hunk 1 above)
Comment (bserdar) [inline crud/src/main/java/com/redhat/lightblue/crud/CRUDOperationContext.java]: Done.

File: crud/src/main/resources/json-schema/response.json (hunk 2 above)
Comment (bserdar) [inline crud/src/main/resources/json-schema/response.json]: This was wrong already. It is "errors", not "otherErrors"

**Pull request merged**